source/WindowScroller/utils/dimensions.js

```diff
@@ -26,5 +26,5 @@ export function getHeight (element) {
  */
 export function getPositionFromTop (element, container) {
   const containerElement = container === window ? document.documentElement : container
-  return element.getBoundingClientRect().top - containerElement.getBoundingClientRect().top
+  return element.getBoundingClientRect().top + getVerticalScroll(container) - containerElement.getBoundingClientRect().top
```

Comment:
@bvaughn Any chance you could look at this sometime soonish? Too long in PRgatory makes me anxious 😅